Jenkinsfile

```diff
@@ -72,6 +72,7 @@ pipeline {
           sh("helm secrets upgrade --dry-run --wait --install --debug  --namespace production --set deployments.live.version=${VERSION.printable()} aws-credentials-broker -f deploy/aws-credentials-broker/secrets.yaml ./deploy/aws-credentials-broker")
           sh("helm secrets upgrade --wait --install --debug  --namespace production --set deployments.live.version=${VERSION.printable()} aws-credentials-broker -f deploy/aws-credentials-broker/secrets.yaml ./deploy/aws-credentials-broker")
         }
+        syncDependencyLibrary()
```

Comment:
aws-credentials-broker is not a scala library, not sure why it's in projectFilters.sc at all